File: Day 4/RockPaperScissors.java
import java.util.Scanner;
import java.util.Arrays;
import java.util.Random;

public class RockPaperScissors {

    public RockPaperScissors() {
        // constructor
    }

// pieces = ["rock", "paper", "scissors"]


    public static void main(String[] args) {
        Scanner userInput = new Scanner(System.in);
        Random random = new Random();
        String[] pieces = {"rock", "paper", "scissors"};
        int randomIndex = random.nextInt(pieces.length);
        String computerChoice = pieces[randomIndex];

        System.out.println("Rock, Paper, Scissors Game");
        System.out.println("==========================");

        System.out.println("Rock, paper, scissors...shoot! (make your choice) ");
        String userChoice = userInput.nextLine();
        userInput.close();


        if (userChoice.equalsIgnoreCase(computerChoice)) { 
            System.out.println("You chose " + userChoice + ".");
            System.out.println("The computer chose " + computerChoice + ".");
            System.out.println("It is a tie!");
            } else if (userChoice.equalsIgnoreCase("rock")){ 
                if (computerChoice.equalsIgnoreCase("paper")){
                    System.out.println("You chose " + userChoice + ".");
                    System.out.println("The computer chose " + computerChoice + ".");
                    System.out.println("You lose!");
                } else {
                    System.out.println("You chose " + userChoice + ".");
                    System.out.println("The computer chose " + computerChoice + ".");
                    System.out.println("You win!");
                }
            } else if (userChoice.equalsIgnoreCase("paper")) {
                if (computerChoice.equalsIgnoreCase("scissors")){
                    System.out.println("You chose " + userChoice + ".");
                    System.out.println("The computer chose " + computerChoice + ".");
                    System.out.println("You lose!");
                } else {
                    System.out.println("You chose " + userChoice + ".");
                    System.out.println("The computer chose " + computerChoice + ".");
                    System.out.println("You win!");
                }
            } else if (userChoice.equalsIgnoreCase("scissors")) {
                if (computerChoice.equalsIgnoreCase("roc")){
                    System.out.println("You chose " + userChoice + ".");
                    System.out.println("The computer chose " + computerChoice + ".");
                    System.out.println("You lose!");
                } else {
                    System.out.println("You chose " + userChoice + ".");
                    System.out.println("The computer chose " + computerChoice + ".");
                    System.out.println("You win!");
                }
            }
        }
    }

File: Day 4/RockPaperScissors.py
import random

pieces = ["rock", "paper", "scissors"]

print("Rock, Paper, Scissors Game")
print("==========================")

userInput = input("Rock, paper, scissors...shoot! (make your choice) ")
computerInput = random.choice(pieces)

if userInput.lower() == computerInput:
    print(f'You chose {userInput}')
    print(f'The computer chose {computerInput}')
    print("It is a tie!")
elif userInput.lower() == "rock":
    if computerInput == "paper":
        print(f'You chose {userInput}')
        print(f'The computer chose {computerInput}')
        print("You lose!")
    else:
        print(f'You chose {userInput}')
        print(f'The computer chose {computerInput}')
        print("You win!")
elif userInput.lower() == "paper":
    if computerInput == "scissors":
        print(f'You chose {userInput}')
        print(f'The computer chose {computerInput}')
        print("You lose!")
    else:
        print(f'You chose {userInput}')
        print(f'The computer chose {computerInput}')
        print("You win!")
elif userInput.lower() == "scissors":
    if computerInput == "rock":
        print(f'You chose {userInput}')
        print(f'The computer chose {computerInput}')
        print("You lose!")
    else:
        print(f'You chose {userInput}')
        print(f'The computer chose {computerInput}')
        print("You win!")